### PR TITLE
completed css-positioning

### PR DESCRIPTION
**Pull request by rachaelkim831**
relative-screenshot
absolute-screenshot
fixed-screenshot

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 175px;
+  left: 25px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
```
Comment (Zhiwutian) [inline css-positioning/absolute.css:33]: you do need to add the proper CSS to make sure the overlay is covering the entire element and darkening it.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed for the overlay

**Comment by rachaelkim831** [comment]
absolute-screenshot

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**